src/__tests__/generateConfluenceMarkdown.test.ts

```diff
@@ -113,39 +111,38 @@ describe('generateConfluenceMarkdown function', () => {
 
         mockDependencyMap.set(dependency, singleIssue);
         const results = mariner.generateConfluenceMarkdown(mockDependencyMap);
-        expect(results).not.toContain(`h3. ${dependency}`);
+        expect(results).not.toContain(dependency);
         expect(results).not.toContain('\n||*Title*||*Age*||');
-        expect(results).not.toContain(
-            `|[${singleIssue[0].title}|${singleIssue[0].url}]|8&nbsp;days|`
-        );
+        expect(results).not.toContain(singleIssue[0].title);
     });
 });
 
 describe('cleanMarkdown function', () => {
     it('should return string without altering it', () => {
         const title1 =
-            '|[2020 upgrade edition menus too narrow|https://github.com/marmelab/react-admin/issues/7520]|8&nbsp;days|';
+            '|2020 upgrade edition menus too narrow|https://github.com/marmelab/react-admin/issues/7520|8&nbsp;days|';
```

Comment:
Hmmmm.....This string is no longer realistic, since it should contain [ ] around the link. Since we will only pass a title to cleanMarkdown, we should probably make this sample test string be just a title. It would probably be best to copy a real issue title (that doesn't have brackets or braces) as a starting point. Then maybe add an & and a | just to show that those don't get changed.